<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SDRReceiver</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="7">
     <widget class="QCustomPlot" name="spectrum" native="true"/>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="startSDR">
      <property name="text">
       <string>Start SDR</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QSpinBox" name="spinBox"/>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="processFile">
      <property name="text">
       <string>Process File</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QComboBox" name="comboVFO">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QComboBox" name="comboSDRs"/>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="stopSDR">
      <property name="text">
       <string>Stop SDR</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="biasTee">
      <property name="text">
       <string>BiasTee</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
